provider/ec2/instance.go

```diff
@@ -119,3 +119,109 @@ func (inst *ec2Instance) IngressRules(ctx context.ProviderCallContext, machineId
 	}
 	return ranges, nil
 }
+
+// AWK SDK version of ec2Instance.
+//
+// TODO(benhoyt) - get rid of ec2Instance and rename this to ec2Instance when
```

Comment:
A little extra churn upfront but my preference would be to rename the other one as legacyInstance or something and start off with the new one being named correctly. Then we just delete the legacy instances as we go and eventually it will all be done with no further churn needed